src/meta/src/rpc/ddl_controller.rs

```diff
@@ -81,6 +82,7 @@ impl StreamingJobId {
     }
 }
 
+pub type RunInBackground = bool;
```

Comment:
I'd suggest making this an `enum`. `Foreground` and `Background` could be much more readable than `true` and `false`.

Reply:
Changing it to `DdlExecutionMode` with the two modes you mentioned.

Open to better names.